crates/jstz_cli/src/bridge/deposit.rs

```diff
@@ -14,18 +16,52 @@ pub fn exec(
 ) -> Result<()> {
     let cfg = Config::load()?;
 
-    let to = to.resolve(&cfg)?;
-    debug!("resolved `to` -> {:?}", to);
+    // Check network
+    if cfg.network_name(&network)? == NetworkName::Dev && cfg.sandbox.is_none() {
+        bail_user_error!(
+            "No sandbox is currently running. Please run {}.",
+            styles::command("jstz sandbox start")
+        );
+    }
 
-    // 2. Execute the octez-client command
-    cfg.octez_client(&network)?.call_contract(
-        &from,
-        "jstz_bridge",
-        "deposit",
-        &format!("(Pair {} 0x{})", amount, hex::encode_upper(to.as_bytes())),
-    )?;
+    let to_pkh = to.resolve(&cfg)?;
+    debug!("resolved `to` -> {:?}", to_pkh);
 
-    info!("Deposited {} CTEZ to {}", amount, to);
+    // Check if trying to deposit to a bootsrap account.
+    if let Some(bootstrap_account) = SANDBOX_BOOTSTRAP_ACCOUNTS
+        .iter()
+        .find(|account| account.address == to_pkh.to_string())
+    {
+        bail_user_error!(
+            "Cannot deposit to the bootstrap account '{}'.",
+            bootstrap_account.address
+        );
+    }
```

Comment:
Not too sure this is required

Reply:
I think it is - it clearly shouldn't be possible to bridge to a bootstrap account. 

We might want to have something where the bridge contract declines it automatically. Anyway, the bridge will be reworked quite a lot in the future - e.g. added withdrawals, tokens etc.